index.html

```diff
@@ -2,10 +2,200 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <a href="#about">About</a>
+          </li>
+          <li>
+            <a href="#work">Work</a>
+          </li>
+          <li>
+            <a href="./images/Resume.pdf" target="blank">Resume</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="intropage">
+      <figure class="profilepic">
+        <img
+          src="./images/Profile1.jpg"
+          height="600px"
+          width="600px"
+          alt="My avatar"
+        />
+      </figure>
+      <aside class="intro">
+        <span id="intro1"> </span><br />
+        <span id="intro2"> </span><br />
+        <span id="intro3"> </span>
+      </aside>
+    </section>
+
+    <section class="icons">
+      <ul>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/madhuri-m-96b89481/"
+            target="blank"
+          >
+            <img src="./images/icons8-linkedin-48.png"
+          /></a>
+        </li>
+        <li>
+          <a href="https://github.com/madhurimt" target="blank">
+            <img src="./images/icons8-github-48.png"
+          /></a>
+        </li>
+        <li>
+          <a href="mailto:madhuri.maniktahla@gmail.com?" target="blank">
```

Comment:
remove ? from the end
```suggestion
          <a href="mailto:madhuri.maniktahla@gmail.com" target="blank">
```